pkg/utils/kube/ctl.go

```diff
@@ -442,30 +441,9 @@ func (k *KubectlCmd) authReconcile(ctx context.Context, config *rest.Config, kub
 	return strings.Join(out, ". "), nil
 }
 
-func Version() (string, error) {
```

Comment:
I've removed this function as it's out of line - it's the only thing that calls `kubectl` and is not used in the engine. I hope that's ok. If argo needs it, it can be copied. I hope you agree with the idea of keeping this library clean and small.